Modules/_testcapimodule.c

```diff
@@ -36,6 +36,9 @@
 #  error "_testcapi must test the public Python C API, not CPython internal C API"
 #endif
 
+/* Ignore use of deprecated APIs */
```

Comment:
Would you mind to give one example of used function which is deprecated? So if tomorrow, the function call is removed, we can also remove this pragma.

Or maybe try to put it around/closer to the deprecated call?